_layouts/production.html

```diff
@@ -2,87 +2,112 @@
 layout: default
 ---
 
+<!-- This bit of Liquid counts the number of images, image_count, for use in flow control below -->
```

Comment:
though the comment is appreciated, please go ahead and delete from your branch.  We can infer what this code does from comments below (show Carousel comment, etc).